draft-ietf-tls-ctls.md

```diff
@@ -512,8 +554,16 @@ which profile.
 
 # IANA Considerations
 
-This document has no IANA actions.
+This document requests that a code point be allocated from the "TLS ContentType
+registry.  This value must be in the range 0-31 (inclusive).  The row to be
+added in the registry has the following form:
+
+| Value | Description | DTLS-OK | Reference |
+|:=====:|:============|:========|:==========|
+|  TBD  | ctls        | N       | RFCXXXX   |
 
+[[ RFC EDITOR: Please replace the value TBD with the value assigned by IANA, and
+the value XXXX to the RFC number assigned for this document. ]]
```

Comment:
I suggest that we require standards action for all profile IDs that would fit in 2 octets.